src/main/main.ml

```diff
@@ -19,19 +19,23 @@ let no_proof = ref false
 
 module S = Msat_sat
 
-module Process() = struct
-  module D = Msat_backend.Dot.Make(S)(Msat_backend.Dot.Default(S))
+module Process () = struct
+  module D = Msat_backend.Dot.Make (S) (Msat_backend.Dot.Default (S))
 
   let hyps = ref []
 
   let st = S.create ~store_proof:(not !no_proof) ~size:`Big ()
 
   let check_model sat =
     let check_clause c =
-      let l = List.map (function a ->
-          Log.debugf 99
-            (fun k -> k "Checking value of %a" S.Formula.pp a);
-          sat.Msat.eval a) c in
+      let l =
+        List.map
+          (function
+            | a ->
```

Comment:
This is a weird formatting... Is it due to the use of 'function' rather than 'fun' ?

Reply:
I think so, that's how I'd format a `function` too. Replacing with `fun` should fix it.